.github/workflows/ci.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
       - name: Run Unit Tests (Electron)
         id: electron-unit-tests
```

Comment:
We really only care about getting code coverage for our stuff - not vs. 

I'm not seeing how changing this fixed the test runs though. Could you explain what's going on here?

Reply:
I made that change after I reviewed how [Darwin](https://github.com/microsoft/azuredatastudio/blob/bc0d66a75ae6b1dd11ecb94157d11068f60324ff/.github/workflows/ci.yml#L215) was running its tests and found that it was using
`run: DISPLAY:10 ./scripts/test.sh` without any command options like `--coverage`. I gave that a try, but kept the command option for `--coverage` to keep my change minimal, but I also don't really see how that fixed it.

Reply:
Yeah, that's on purpose since we only need to collect coverage for one run. And as mentioned we shouldn't be collecting code coverage for VS stuff since that would make the number mostly useless for determining coverage of our stuff.

So it seems like there's a VS test that's doing something that causes the animation stuff to start working properly. If you run `./scripts/test.(bat|sh) --runGlob "**/sql/**/*.test.js"` locally does it pass?

Reply:
Yeah, those test fail with --runGlob "**/sql/**/*.test.js"
![image](https://user-images.githubusercontent.com/87730006/135510833-45780ca2-d829-441c-a2ce-f56d89292832.png)

Reply:
It passes without:
![image](https://user-images.githubusercontent.com/87730006/135511894-97ae66eb-fc4d-4d9f-b24c-fc37a5e65911.png)

Reply:
So then I would suggest trying to track down what test it is that's causing it to be able to pass. So doing something like a binary search where you run the scrollable view tests + half of the VS tests and then keep splitting that down until you find which test(s) cause it to pass. Ideally then there would be some code that the test is doing which you could copy over for our test to also run even if none of the VS tests are running.